### PR TITLE
Add an resources for github-affecting workflows

### PR DESCRIPTION
**Pull request by phalbert**
Workflow for this [guide](https://github.com/port-labs/port-docs/pull/892)

### CODE REVIEW

**Review by yairsimantov20 — APPROVED** [review]
🌊

**Review by yairsimantov20 — CHANGES_REQUESTED** [review]
left a few comments

**Review by yairsimantov20 — CHANGES_REQUESTED** [review]
Left comments

File: github-workflows/aws/tag-aws-cli.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_REGION }}
+
+    - name: Install jq
+      run: sudo apt-get install jq
```
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/aws/tag-aws-cli.yml:26]: alright, thank you for pointing this out.

File: github-workflows/aws/tag-aws-cli.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - name: Add Tags to S3 Bucket
+      env:
+        BUCKET_NAME: ${{ fromJson(inputs.port_payload).context.entity }}
+        TAGS_JSON: ${{ github.event.inputs.tags }}
+      run: |
+        # Extract key-value pairs from the JSON object
+        # {
+        #   "env": "test",
+        #   "team": "beta"
+        # }
```
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/aws/tag-aws-cli.yml:37]: Yes, I believe including a sample from the JSON object here can be beneficial for preemptively addressing potential questions about the expected object.

File: github-workflows/github/delete-repo.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      
+      - name: Inform completion of deletion
+        uses: port-labs/port-github-action@v1
+        with:
+          clientId: ${{ secrets.PORT_CLIENT_ID }}
+          clientSecret: ${{ secrets.PORT_CLIENT_SECRET }}
+          operation: PATCH_RUN
+          runId: ${{ fromJson(inputs.port_payload).context.runId }}
+          logMessage: |
+            GitHub repository deleted! ✅
```
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/github/delete-repo.yml]: handled it now

File: github-workflows/github/manage-pr.yml
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/github/manage-pr.yml:1]: added these now

File: github-workflows/aws/tag-aws-cli.yml
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/aws/tag-aws-cli.yml:1]: Added this now

File: github-workflows/aws/tag-aws-cli.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+on:
+  workflow_dispatch:
+    inputs:
+      tags: # json object
+        required: true
+        type: string
+      port_payload:
+        required: true
+        description: "Port's payload, including details for who triggered the action and general context (blueprint, run id, etc...)"
+        type: string
```
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/aws/tag-aws-cli.yml:12]: what do you mean here? I thought the `required: true` is the validation.

File: github-workflows/azure/tag-azure-cli.yml
[the remark this answers is not in the document]
Comment (phalbert) [reply github-workflows/azure/tag-azure-cli.yml:1]: alright

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Please apply requested changes and comments from me and Yair and let me know when done

**Comment by phalbert** [comment]
I have moved these actions to port-docs:

1. [Manage GitHub PRs](https://github.com/port-labs/port-docs/pull/1029)
2. [Add Tags to S3 bucket](https://github.com/port-labs/port-docs/pull/1031)
3. [Tag Azure Resource](https://github.com/port-labs/port-docs/pull/1030)

**Pull request closed without merging**